tasks/main.yml

```diff
@@ -98,6 +106,39 @@
   until: apt_getresult is succeeded
   retries: "{{ bootstrap_retries }}"
 
+- name: check if packages are already installed
```

Comment:
I'm not sure how, but I'd like this to be one step, maybe:

```
- name: install software with pkg_add
  raw: pkg_add -Iz {{ bootstrap_pkg_add_packages }}
  register: pkg_add_getresult
  changed_when:
    - "': ok' in pkg_add_getresult.stdout"
  remote_user: "{{ bootstrap_user }}"
  until: pkg_add_getresult is succeeded
  retries: "{{ bootstrap_retries }}"
```

So that basically means it will always (attempt to) install, but ansible will show "changed" when something happened.

Reply:
You're right, actually I added check to achieve idempotency, but one step works well.